README.md

```diff
@@ -185,30 +185,30 @@ to a version of your choice.
 ```sh
 # To roll back a client to specific version, you need to specify which version do you want to install.
 # Format of a flag is --<client>-tag, e.g. for Prysm:
-$ lukso install --prysm-tag v4.2.1
+$ lukso install --prysm-tag v5.0.4
 # or Erigon:
 $ lukso install --erigon-tag 2.55.0
 
 # When rolling back Geth, please make sure to also provide first 8 characters of release commit hash.
 # You can find those on Geth's download page: https://geth.ethereum.org/downloads
-$ lukso install --geth-tag 1.13.15 --geth-commit-hash c5ba367e
+$ lukso install --geth-tag 1.14.7 --geth-commit-hash c5ba367e
```

Comment:
`--geth-commit-hash` stays the same?

Reply:
While this is just a command example, youre right, this is not the correct commit hash for this geth version - just fixed this